src/main/resources/templates/db_scripts/create_records_orders_table.sql

```diff
@@ -0,0 +1,5 @@
+-- Creates table to store association between record id and order id
+CREATE TABLE IF NOT EXISTS records_orders (
```

Comment:
is record here a Kafka record?

does this table get cleaned out sometimes?

Reply:
here it means an srs record. For now, this table is not cleaned.